project.clj

```diff
@@ -31,12 +31,8 @@
   :profiles {:dev {:plugins      [[com.jakemccrary/lein-test-refresh "0.24.1"]
                                   [lein-ancient "0.6.15"]
                                   [lein-auto "0.1.3"]
-                                  [lein-set-version "0.4.1"]
-                                  [venantius/ultra "0.6.0"]]
-                   :dependencies [[circleci/circleci.test "0.4.2"]
-                                  [pjstadig/humane-test-output "0.9.0"]]
-                   :injections   [(require 'pjstadig.humane-test-output)
-                                  (pjstadig.humane-test-output/activate!)]}}
```

Comment:
Should be activated by user profile

https://github.com/pjstadig/humane-test-output#clojure

Also doesn't work with latest cljs

Reply:
Is not compatible with Cursive, and they recommend disabling when running cursive (through env var)